etc/ci/genindex.py

```diff
@@ -0,0 +1,956 @@
+#!/usr/bin/env python3
+# ---
+# https://github.com/glowinthedark/index-html-generator/
```

Comment:
Replace the link with: https://github.com/glowinthedark/index-html-generator/tree/639db17b0a5856f1c4060826c83aebba2e6cc9dc

Reply:
Done!